examples/pipelines/pneumonia/submit.py

```diff
@@ -0,0 +1,302 @@
+from omegaconf import OmegaConf
+
+
+############################
+### CONFIGURE THE SCRIPT ###
+############################
+
+parser = argparse.ArgumentParser(description=__doc__)
+
+parser.add_argument(
+    "--config",
+    type=str,
+    required=False,
+    default=os.path.join(os.path.dirname(__file__), "config.yaml"),
+    help="path to a config yaml file",
+)
+parser.add_argument(
+    "--submit",
+    default=False,
+    action="store_true",
+    help="actually submits the experiment to AzureML",
+)
+
+parser.add_argument(
+    "--subscription_id",
+    type=str,
+    required=False,
+    help="Subscription ID",
+)
+parser.add_argument(
+    "--resource_group",
+    type=str,
+    required=False,
+    help="Resource group name",
+)
+
+parser.add_argument(
+    "--workspace_name",
+    type=str,
+    required=False,
+    help="Workspace name",
+)
+
+parser.add_argument(
+    "--wait",
+    default=False,
+    action="store_true",
+    help="Wait for the pipeline to complete",
+)
+
+args = parser.parse_args()
+
+# load the config from a local yaml file
+YAML_CONFIG = OmegaConf.load(args.config)
+
+# path to the components
+COMPONENTS_FOLDER = os.path.join(
+    os.path.dirname(__file__), "..", "..", "components", "PNEUMONIA"
+)
+
+
+###########################
+### CONNECT TO AZURE ML ###
+###########################
+
+
+def connect_to_aml():
+    try:
+        credential = DefaultAzureCredential()
+        # Check if given credential can get token successfully.
+        credential.get_token("https://management.azure.com/.default")
+    except Exception as ex:
+        # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential does not work
+        credential = InteractiveBrowserCredential()
+
+    # Get a handle to workspace
+    try:
+        # tries to connect using local config.json
+        ML_CLIENT = MLClient.from_config(credential=credential)
+
+    except Exception as ex:
+        print(
+            "Could not find config.json, using config.yaml refs to Azure ML workspace instead."
+        )
+
+        # tries to connect using cli args if provided else using config.yaml
+        ML_CLIENT = MLClient(
+            subscription_id=args.subscription_id or YAML_CONFIG.aml.subscription_id,
+            resource_group_name=args.resource_group
+            or YAML_CONFIG.aml.resource_group_name,
+            workspace_name=args.workspace_name or YAML_CONFIG.aml.workspace_name,
+            credential=credential,
+        )
+    return ML_CLIENT
+
+
+ML_CLIENT = connect_to_aml()
+
+####################################
+### LOAD THE PIPELINE COMPONENTS ###
+####################################
+
+# Loading the component from their yaml specifications
+training_component = load_component(
+    source=os.path.join(COMPONENTS_FOLDER, "traininsilo", "spec.yaml")
+)
+
+aggregate_component = load_component(
+    source=os.path.join(
+        COMPONENTS_FOLDER, "aggregatemodelweights", "spec.yaml"
```

Comment:
What I meant this morning is that it would be easy to divert just this line to call the generic aggregatemodelweights instead. But up to you, as long as we modify this before the release.